COMPILATION.md

```diff
@@ -141,30 +141,19 @@ loaded.
 ### Prerequisites
 
 - msys2 installation with MinGW64 C++ compiler 
-  - Get the installer directly from the msys2 homepage https://www.msys2.org/ (use msys2-x86_64.exe for Windows 64-bit)
+  - Get the installer directly from the msys2 homepage https://www.msys2.org/ (use msys2-x86_64-.exe for Windows 64-bit)
```

Comment:
Last time I suggested to use msys2-x86_64*.exe with a * wildcard. You convinced me that computer users these days do not know filename wildcards anymore and we agreed to use a workaround description like "(use the installer whose name starts with "msys2-x86_64" for 64 bit windows)".

The dash that is here now is unexpected. Is this your latest version?